src/Views/gagepage.html

```diff
@@ -225,13 +225,27 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                         &nbsp;<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>
                     </div>
                     <center>
-                    <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.chartConfig">
-                        <input type="checkbox" ng-if="vm.formattedFloodFreq.length > 0" id="yesAEPLines" name="yesAEPLines" ng-model="vm.plotlines" ng-change="vm.togglePlotLines()" checked>
-                        <label for="yesAEPLines" ng-if="vm.formattedFloodFreq.length > 0"> AEP Plot Lines&nbsp;&nbsp;</label>
-                        <input type="checkbox" ng-if="vm.formattedDailyFlow.length > 0" id="logScale" name="logScale" ng-model="vm.logScale" ng-change="vm.toggleLogLinear()" checked>
-                        <label for="logScale" ng-if="vm.formattedDailyFlow.length > 0"> Log Scale</label>
-                        <highchart id="chart1" config="vm.chartConfig"></highchart>
-                    </div>
+                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.chartConfig">
+                            <div class="plot-filters">
+                                <input type="checkbox" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0" id="toggleFloodStats" name="toggleFloodStats" ng-model="vm.showFloodStats" ng-change="vm.toggleFloodStats()" checked>
+                                <label for="toggleFloodStats" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0" > Show Flood Statistics:</label>
+                                <select id="gageFloodStatsSelect" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0 " style="width:10%" name="gageFloodStatsSelect" ng-model="vm.selectedFloodFreqStats" ng-options="floodFreqStat.name for floodFreqStat in vm.allFloodFreqStats" ng-change="vm.chooseFloodStats()" ng-disabled="vm.showFloodStats == false"></select>&nbsp;&nbsp;&nbsp;                      
```

Comment:
Is it possible to reformat this area so that the select box can be wide enough to show the full name of the Statistic Group? Maybe that checkbox and select box just needs its own line.
![image](https://user-images.githubusercontent.com/40237491/233503348-7e6e8a18-1188-49bb-9f79-067e9f470cdd.png)


```suggestion
                                <label for="toggleFloodStats" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0" > Show Flood Statistics:</label>
                                <select id="gageFloodStatsSelect" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0 " style="width:10%" name="gageFloodStatsSelect" ng-model="vm.selectedFloodFreqStats" ng-options="floodFreqStat.name for floodFreqStat in vm.allFloodFreqStats" ng-change="vm.chooseFloodStats()" ng-disabled="vm.showFloodStats == false"></select>&nbsp;&nbsp;&nbsp;                      
```

Reply:
I made it longer so that "Annual Exceedance Probability" fits in full, but for gages that have alternate stats it's just long enough to show "Alternative Annual Exceedance". All the other stats (1-Day Low Flow, etc.) are much shorter. I didn't want it to be too long to the point where it takes up a ton of space, but let me know what you think, though, and it can definitely be even longer.